frontend/src/modules/organization/components/details/identity/organization-details-identity-item.vue

```diff
@@ -0,0 +1,204 @@
+      <lf-tooltip
+        v-else
+        placement="top"
+        :disabled="!isVerifyDisabled"
+        class="!w-full"
+      >
+        <template #content>
+          Identities tracked from an Integrations<br> can’t be verified
+        </template>
+        <lf-dropdown-item
+          :disabled="isVerifyDisabled"
+          class="w-full"
+          @click="verifyIdentity(true)"
+        >
+          <lf-icon name="verified-badge-line" />Verify identity
+        </lf-dropdown-item>
+      </lf-tooltip>
+
+      <!-- Unmerge -->
+      <lf-dropdown-item @click="emit('unmerge')">
+        <lf-icon name="link-unlink" />Unmerge identity
+      </lf-dropdown-item>
+
+      <lf-dropdown-separator />
+      <lf-dropdown-item
+        type="danger"
+        class="w-full"
+        @click="removeIdentity"
+      >
+        <lf-icon name="delete-bin-6-line" />Delete identity
+      </lf-dropdown-item>
+    </lf-dropdown>
+  </article>
+</template>
+
+<script setup lang="ts">
+import LfIcon from '@/ui-kit/icon/Icon.vue';
+import { CrowdIntegrations } from '@/integrations/integrations-config';
+import LfTooltip from '@/ui-kit/tooltip/Tooltip.vue';
+import LfButton from '@/ui-kit/button/Button.vue';
+import LfDropdown from '@/ui-kit/dropdown/Dropdown.vue';
+import LfDropdownItem from '@/ui-kit/dropdown/DropdownItem.vue';
+import LfDropdownSeparator from '@/ui-kit/dropdown/DropdownSeparator.vue';
+import Message from '@/shared/message/message';
+import LfSvg from '@/shared/svg/svg.vue';
+import usePermissions from '@/shared/modules/permissions/helpers/usePermissions';
+import { LfPermission } from '@/shared/modules/permissions/types/Permissions';
+import { computed, ref } from 'vue';
+import { Organization, OrganizationIdentity } from '@/modules/organization/types/Organization';
+import { useOrganizationStore } from '@/modules/organization/store/pinia';
+
+const props = defineProps<{
+  identity: OrganizationIdentity,
+  organization: Organization
+}>();
+
+const emit = defineEmits<{(e: 'edit'): void, (e: 'unmerge'): void }>();
+
+const { hasPermission } = usePermissions();
+
+const { updateOrganization } = useOrganizationStore();
+
+const hovered = ref(false);
+
+const platform = (name: string) => CrowdIntegrations.getConfig(name);
+
+const isVerifyDisabled = computed(
+  () => !!props.identity.sourceId || ['integration', 'lfid'].includes(props.identity.platform),
+);
+
+const verifyIdentity = (verified: boolean) => {
+  const identities = props.organization.identities.map((i: OrganizationIdentity) => {
+    if (i.platform === props.identity?.platform && i.value === props.identity?.value) {
+      return {
+        ...i,
+        verified,
+      };
+    }
+    return i;
+  });
+
+  updateOrganization(props.organization.id, {
+    identities,
+  })
+    .catch(() => {
+      Message.error('Something went wrong while updating an identity');
+    })
+    .then(() => {
+      Message.success('Identity updated successfully');
+    });
+};
```

Comment:
If you try to verify the github identity https://lf-staging.crowd.dev/organizations/35fb26b0-da1e-11ee-85d4-778cfceb14e0?projectGroup=901e303b-5188-4338-9be8-dd96f45d7eca&segmentId=901e303b-5188-4338-9be8-dd96f45d7eca

API is throwing an error. However, frontend displays both the error and success message. Can you check what's wrong with the toast messages?